manual/query_builder/relation/README.md

```diff
@@ -0,0 +1,203 @@
+## Relations
+
+A relation is a clause that appears after the WHERE keyword, and restricts the rows that the
+statement operates on.
+
+Relations are used by the following statements:
+
+* [SELECT](../select/) 
+* [UPDATE](../update/)
+* [DELETE](../delete/)
+
+To create a relation, call one of the `isXxx()` methods of `QueryBuilderDsl`, chain it with one of
+the available "operator" methods, and pass the result to `where()`:
+
+```java
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilderDsl.*;
+
+Relation idRelation = isColumn("id").eq(bindMarker());
+selectFrom("user").all().where(idRelation);
+// SELECT * FROM user WHERE id=?
+```
+
+If you call `where()` multiple times, the clauses will be joined with the AND keyword. You can also
+add multiple relations in a single call. This is a bit more efficient since it creates less
+temporary objects: 
+
+```java
+selectFrom("sensor_data").all()
+    .where(
+        isColumn("id").eq(bindMarker()),
+        isColumn("date").gt(bindMarker()));
+// SELECT * FROM sensor_data WHERE id=? AND date>?
+```
+
+Relations are generally composed of a left-hand-side target, an operator, and an optional
+right-hand-side [term](../term/). The type of relation determines which operators are available.
+ 
+### Simple columns
+
+`isColumn` operates on a single column. It supports basic arithmetic comparison operators:
+
+| Comparison operator | Method name |
```

Comment:
One thought,  we use shorthand method names for comparison operators, but not arithmetic operations.  Should we use same strategy for both?

Reply:
I copied over the 3.x names without thinking about it, but you're right, the API should be consistent.
There are no obvious abbreviations for the arithmetic operators, and I dislike abbreviations anyway, so I think we should rename the comparison ones (taking inspiration from the AssertJ syntax): `equalTo`, `lessThan`, `lessThanOrEqualTo`, `greaterThan`, `greaterThanOrEqualTo`, `notEqualTo`. Some of them are a bit verbose, but they have the merit to be unambiguous.